static/app/components/profiling/flamegraph/flamegraph.tsx

```diff
@@ -196,6 +197,14 @@ function Flamegraph(): ReactElement {
     );
   }, [organization.features, profileGroup.metadata.platform]);
 
+  const hasCPUChart = useMemo(() => {
+    const platform = profileGroup.metadata.platform;
+    return (
+      (platform === 'cocoa' || platform === 'android') &&
```

Comment:
Should we start making a list of platform features somewhere? Like

```typescript
const PROFILING_FEATURES = {
  'chart.cpu': new Set(['cocoa', 'android']),
};
```

Reply:
Yup, let me get back to this, but agree we should have something for this. Right now that could just be `isMobile` helper tbh. I'll take the opportunity to refactor when we have memory chart in there too